compiler/rustc_metadata/src/rmeta/mod.rs

```diff
@@ -161,7 +148,7 @@ enum LazyState {
 
     /// Inside a metadata node, with a previous `Lazy`.
     /// The position is a conservative estimate of where that
-    /// previous `Lazy` would end (see their comments).
+    /// previous `Lazy` would start (see their comments).
```

Comment:
You can remove the "conservative estimate", which only made sense with `min_size`.

Reply:
Amended the commit.